packages/block-editor/src/components/index.js

```diff
@@ -60,6 +60,8 @@ export { default as withColorContext } from './color-palette/with-color-context'
 
 export { default as __experimentalBlockSettingsMenuFirstItem } from './block-settings-menu/block-settings-menu-first-item';
 export { default as __experimentalInserterMenuExtension } from './inserter-menu-extension';
+export { default as __experimentalPreviewOptions } from './preview-options';
+export { default as __experimentalUseResizeCanvas } from './resize-canvas';
```

Comment:
What's the reasoning for including these components and their state in the block-editor package?

Reply:
I was just looking for a good place for them so that they could be used both in `edit-post` and `edit-site` context. I didn't want to place them in `editor` since it depends on wp and post, and this could be useful outside of that context too. Do you think they should be placed in some other package?

Reply:
@youknowriad based on our conversation I removed the required state from `block-editor` component and placed it in `edit-post` and `edit-site` separately in dc50eb9cb1d3f60414aab4c4aab30f3ec97dc3b4.

I considered using local state for it, but since both `PreviewOptions` and `useResizeCanvas` rely on it, and these are at different nesting levels and parts of the editor, this seemed cleaner that propagating this through props or using context.